### PR TITLE
fix/CHAT-784

### PR DESCRIPTION
**Pull request by pnedonosko**
Fix includes URL build without RFC validation and improved WebRTC call log levels (for remote log).

### CODE REVIEW

File: webrtc/services/src/main/java/org/exoplatform/webconferencing/webrtc/server/WebrtcCallFilter.java [hunk 1]
```diff
@@ -82,15 +84,19 @@ public void doFilter(ServletRequest request, ServletResponse response, FilterCha
...
-          URI secureURI = new URI(SCHEME_HTTPS,
-                                  null,
+          // TODO
+          // URI secureURI = new URI(SCHEME_HTTPS,
+          // null,
+          // httpReq.getServerName(),
+          // httpReq.getServerPort(),
+          // httpReq.getServletPath(),
+          // httpReq.getQueryString(),
+          // null);
```
Comment (thomasdelhomenie) [inline webrtc/services/src/main/java/org/exoplatform/webconferencing/webrtc/server/WebrtcCallFilter.java]: To be removed.

File: webrtc/webapp/src/main/java/org/exoplatform/webconferencing/webrtc/portlet/WebrtcProviderPortlet.java [hunk 2]
```diff
@@ -79,14 +79,21 @@ public void init() throws PortletException {
...
-                              null);
-        Settings settings = provider.settings().callUri(callURI.toString()).locale(request.getLocale()).build();
+        // TODO cleanup
+        // URI callURI = new URI(request.getScheme(),
+        // null,
+        // request.getServerName(),
+        // request.getServerPort(),
+        // "/webrtc/call",
+        // null,
+        // null);
```
Comment (thomasdelhomenie) [inline webrtc/webapp/src/main/java/org/exoplatform/webconferencing/webrtc/portlet/WebrtcProviderPortlet.java]: To be removed.

File: webrtc/services/src/main/java/org/exoplatform/webconferencing/webrtc/server/WebrtcCallFilter.java (hunk 1 above)
Comment (pnedonosko) [reply webrtc/services/src/main/java/org/exoplatform/webconferencing/webrtc/server/WebrtcCallFilter.java]: yep, forget about them

File: webrtc/webapp/src/main/java/org/exoplatform/webconferencing/webrtc/portlet/WebrtcProviderPortlet.java (hunk 2 above)
Comment (pnedonosko) [reply webrtc/webapp/src/main/java/org/exoplatform/webconferencing/webrtc/portlet/WebrtcProviderPortlet.java]: done

**Pull request merged**